ee/clickhouse/queries/funnels/base.py

```diff
@@ -459,7 +460,9 @@ def _build_filters(self, entity: Entity, index: int) -> str:
             team_id=self._team.pk,
             property_group=entity.property_groups,
             prepend=str(index),
-            person_properties_mode=PersonPropertiesMode.USING_PERSON_PROPERTIES_COLUMN,
+            person_properties_mode=PersonPropertiesMode.DIRECT_ON_EVENTS
```

Comment:
`_build_step_query` -> `format_action_filter` requires a similar treatment, I think?

Reply:
good point.. not sure why tests didn't fail from this 😬. going to add more tests